ft_volumelookup.m

```diff
@@ -37,15 +37,17 @@
 %   cfg.maskparameter       = string, field in volume to be looked up, data in field should be logical
 %   cfg.maxqueryrange       = number, should be odd (default = 1)
 %
-% The configuration options for labels around a point of interest:
+% The configuration options for labels around POI:
 %   cfg.output              = 'label'
-%   cfg.roi                 = Nx3 vector, coordinates of the points of interest
+%   cfg.roi                 = Nx3 vector, coordinates of the POI
 %   cfg.inputcoord          = 'mni' or 'tal', coordinate system of the mri/source/stat
 %   cfg.atlas               = string, filename of atlas to use, see FT_READ_ATLAS
 %   cfg.maxqueryrange       = number, should be 1, 3, 5 (default = 1)
-%   cfg.querymethod         = 'sphere' searches voxels around the roi in a sphere (default)
-%                           = 'cube' searches voxels around the roi in a sphere
-%   cfg.round2nearestvoxel = 'yes' or 'no', voxel closest to point of interest is calculated (default = 'yes')
+%   cfg.querymethod         = 'sphere' searches voxels around the ROI in a sphere (default)
+%                           = 'cube' searches voxels around the ROI in a cube
+%   cfg.round2nearestvoxel  = 'yes' or 'no', voxel closest to POI is calculated (default = 'yes')
+%   cfg.multioutput         = 'no' always outputs one label; if several POI are provided, they are considered together as describing a ROI (default)
```

Comment:
I propose you delete the cfg.multioutput option and rather use cfg.output. It now only has one option, which is silly, but apparently was meant for use cases like this. It could have options cfg.output=single/multiple, where ft_checkconfig with renamedval would be used to replace 'label' by 'single' .

Reply:
so your new functionality would be the result of cfg.output=multiple.